File: CalWatchApi/CalWatch.Api/Models/ReportDay.cs
using System;

namespace CalWatch.Api.Models
{
    public class ReportDay
    {
        /// <summary>
        /// Date (day) of this report record.
        /// </summary>
        public DateTime Date { get; set; }
        /// <summary>
        /// Total calories consumed on the given day, in the given time period.
        /// </summary>
        public long TotalCaloriesForPeriod { get; set; }
        public long TotalCaloriesForDay { get; set; }
        /// <summary>
        /// Gets/sets a value indicating whether calories for today exceeded 
        /// daily target. 
        /// </summary>
        public bool OverDailyTarget { get; set; }
    }
}

File: CalWatchAndroid/app/src/main/java/com/calwatch/android/api/models/ReportDay.java
package com.calwatch.android.api.models;

/**
 * Created by John R. Kosinski on 30/1/2559.
 */
public class ReportDay {
    private String date;
    private int totalCaloriesForPeriod;
    private int totalCaloriesForDay;
    private boolean overDailyTarget;

    public String getDate() {
        return date;
    }
    public void setDate(String date) {
        this.date = date;
    }

    public int getTotalCaloriesForPeriod() {
        return totalCaloriesForPeriod;
    }
    public void setTotalCaloriesForPeriod(int totalCaloriesForPeriod) {
        this.totalCaloriesForPeriod = totalCaloriesForPeriod;
    }

    public int getTotalCaloriesForDay() {
        return totalCaloriesForDay;
    }
    public void setTotalCaloriesForDay(int totalCaloriesForDay) {
        this.totalCaloriesForDay = totalCaloriesForDay;
    }

    public boolean getOverDailyTarget() {
        return overDailyTarget;
    }
    public void setOverDailyTarget(boolean overDailyTarget) {
        this.overDailyTarget = overDailyTarget;
    }
}
